src/aspire/utils/preprocess.py

```diff
@@ -302,3 +303,34 @@ def vol2img(volume, rots, L=None, dtype=None):
         im = im * m_reshape(np.exp(2*np.pi*1j*(grid2d['x'] +grid2d['y'])/(2*lv)), (lv, lv, 1))
 
     return np.real(im)
+
+
+def fuzzy_mask(L, r0, risetime, origin=None):
+    """
+    Create a centered 1D to 3D object of radius r0
+
+    Made with an error function with effective rise time.
+    :param L: The size of image
+    :param r0: The specified radius
+    :param risetime: The rise time for `erf` function
+    :param origin: The coordinates of origin
+    :return: The desired fuzzy mask
+    """
+    if isinstance(L, int):
+        L = np.array(L)
+    if isinstance(r0, int):
+        r0 = np.array(r0)
+
+    center = np.floor(L / 2)
+    if origin is None:
+        origin = center
+        origin = origin.astype('int')
+
+    grids = [np.arange(1-org, ell - org + 1) for ell, org in zip(L, origin)]
+    XYZ = np.meshgrid(*grids, indexing='ij')
+    XYZ_sq = [X ** 2 for X in XYZ]
```

Comment:
cool, so we wrote this line to work with k dimensions (a list of k)..

Reply:
This is from Joakim's suggestion.

Reply:
This line was fine, I was noting that is was inconsistent with the following line.